### PR TITLE
Cleanup/menu

### PR DESCRIPTION
**Pull request by Kelvin120**
Changed the color of the selected option and added MenuOptionReset to all options

### CODE REVIEW

File: Festivity/Menu/Menu.cs [hunk 1]
```diff
@@ -15,8 +15,7 @@ public static void Draw(List<MenuOption> consoleOptions)
             {
                 if (Option == i)
                 {
-                    Console.ForegroundColor = ConsoleColor.White;
-                    Console.BackgroundColor = ConsoleColor.DarkGray;
+                    Console.ForegroundColor = ConsoleColor.Blue;
```
Comment (robinfredriks) [inline Festivity/Menu/Menu.cs:18]: Waarom blauw?

Comment (EnnyGit) [reply Festivity/Menu/Menu.cs:18]: Inderdaad

**Review by EnnyGit — APPROVED** [review]

File: Festivity/Menu/Menu.cs (hunk 1 above)
Comment (Kelvin120) [reply Festivity/Menu/Menu.cs:18]: Welke dan

Comment (Samsabal) [reply Festivity/Menu/Menu.cs:18]: Nou geen blauw!

Comment (EnnyGit) [reply Festivity/Menu/Menu.cs:18]: Donkerblauw

Comment (EnnyGit) [reply Festivity/Menu/Menu.cs:18]: Rats Kleur

Comment (Kelvin120) [reply Festivity/Menu/Menu.cs:18]: RATS RATS WE ARE THE RATS RATS RATS WE ARE THE RATS RATS RATS WE ARE THE RATS RATS RATS WE ARE THE RATS RATS RATS WE ARE THE RATS RATS RATS WE ARE THE RATS

**Review by robinfredriks — APPROVED** [review]

**Pull request merged**